specification/applicationinsights/ApplicationInsights.LiveMetrics/models.tsp

```diff
@@ -0,0 +1,567 @@
+import "@typespec/rest";
+import "@typespec/http";
+import "@typespec/openapi";
+import "@azure-tools/typespec-azure-core";
+
+using Azure.Core;
+using TypeSpec.Rest;
+using TypeSpec.Http;
+using TypeSpec.Versioning;
+using OpenAPI;
+
+namespace LiveMetrics;
+
+/**
+ * QuickPulse /ping request parameters, either in query string, or in request body
+ */
+model PingRequest {
+  ...PingRequestQueryStrings;
+  ...PingRequestHeaders;
+  ...PingRequestBody;
+}
+
+/**
+ * QuickPulse ping request query strings
+ */
+model PingRequestQueryStrings {
+  @doc("The ikey of the target Application Insights component for which client sdk calls /ping.")
+  @query
+  ikey: string;
+
+  @doc("Deprecated. An alternative way to pass api key. Use AAD auth instead.")
+  @query
+  apiKey?: string;
+}
+
+/**
+ * QuickPulse ping request headers
+ */
+model PingRequestHeaders {
+  @doc("Timestamp when SDK transmits the metrics and documents to QuickPulse. A 8-byte long type of ticks.")
+  @header("x-ms-qps-transmission-time")
+  transmissionTime?: int64;
+
+  @doc("Computer name where AI SDK lives. QuickPulse uses machine name with instance name as a backup.")
+  @header("x-ms-qps-machine-name")
+  machineName?: string;
+
+  @doc("Service instance name where AI SDK lives. QuickPulse uses machine name with instance name as a backup.")
+  @header("x-ms-qps-instance-name")
+  instanceName?: string;
+
+  @doc("Identifies an AI SDK as trusted agent to report metrics and documents.")
+  @header("x-ms-qps-stream-id")
+  streamId?: string;
+
+  @doc("Cloud role name of the service.")
+  @header("x-ms-qps-role-name")
+  roleName?: string;
+
+  @doc("Version/generation of the data contract (MonitoringDataPoint) between SDK and QuickPulse.")
+  @header("x-ms-qps-invariant-version")
+  invariantVersion?: string;
+
+  @doc("An encoded string that indicates whether the collection configuration is changed.")
+  @header("x-ms-qps-configuration-etag")
+  configurationEtag?: string;
```

Comment:
I don't know if it'll matter much for a header, but it may be better to declare this as type https://azure.github.io/typespec-azure/docs/libraries/azure-core/reference/data-types#Azure.Core.eTag which some language SDKs may use to constrain the acceptable values.

Reply:
Same for other etags below (I see at least one more, but won't repeat this comment).